scripts/gui/microbe_editor.mjs

```diff
@@ -818,15 +830,14 @@ function processPatchMapData(data){
         element.classList.add("PatchContainer");
         element.style.left = patch.screenCoordinates.x + "px";
         element.style.top = patch.screenCoordinates.y + "px";
-
+        element.id = patch.id;
```

Comment:
I think some kind of prefix should be added in order to avoid conflicts, as currently I think this is just a plain number like 0 and might later cause issues. Something like `"patchMapPatch_" + patch.id`